src/services/esri-feature-service.js

```diff
@@ -0,0 +1,26 @@
+import { loadModules } from 'esri-loader';
+
+function getFeatures({ url, whereClause = "", outFields = ["*"], returnGeometry = false, outSpatialReference = 102100 }) {
+  return new Promise((resolve, reject) => {
```

Comment:
Maybe, in the future, we could add SWR if the response is not cached

Reply:
You're right :+1: 
Gonna give it a quick try